state/mysql/mysql.go

```diff
@@ -322,8 +322,8 @@ func schemaExists(db *sql.DB, schemaName string, timeout time.Duration) (bool, e
 	return exists == 1, err
 }
 
-func tableExists(db *sql.DB, tableName string, timeout time.Duration) (bool, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+func tableExists(ctx context.Context, db *sql.DB, tableName string, timeout time.Duration) (bool, error) {
+	ctx, cancel := context.WithTimeout(ctx, timeout)
```

Comment:
Don't override the context but use a different name

Reply:
Same as above. Just helps preventing psosbile future mistakes :) (I know a person who would probably make those mistakes - me :D )